keras_cv/layers/object_detection/roi_sampler_test.py

```diff
@@ -62,57 +62,58 @@ def test_roi_sampler(self):
             np.min(sampled_gt_classes),
         )
 
-    def test_roi_sampler_small_threshold(self):
-        box_matcher = BoxMatcher(thresholds=[0.1], match_values=[-1, 1])
```

Comment:
Please add `self.skipTest("TODO: type message and issue number here")` right before  `box_matcher = BoxMatcher(thresholds=[0.1], match_values=[-1, 1])`

Reply:
Got it! Thanks for the help.